templates/scss-variables.js

```diff
@@ -1,3 +1,7 @@
-module.exports = answers => {
-	return `$${answers.name}-is-silent: true !default;\n`;
+module.exports = () => {
+	// No default Sass variables, but it's common to
+	// use variables.scss for global variables and
+	// useful to have included by default ready to
+	// add variables.
```

Comment:
It might be nicer for the user if we add this as a sass comment into the file, that way they can see what the intended use of the `variables.scss` file is.